src/wasm/extract.cpp

```diff
@@ -31,15 +32,15 @@ file_output::file_output(const fs::path& dir, const processed_file* f, bool writ
       total_written_(0),
       write_(write),
       zip_(zip),
-      zip_open_(false) {
+      file_open_(false) {
   if (write_) {
     try {
       std::ios_base::openmode flags =
           std::ios_base::out | std::ios_base::binary | std::ios_base::trunc;
       if (file_->is_multipart()) {
         flags |= std::ios_base::in;
       }
-      if(zip_.getStatus() != NONZIP_STATUS_READY) {
+      if(zip_ == NULL) {
```

Comment:
Missing space before parenthesis.
```suggestion
      if (zip_ == NULL) {
```